common/post-fs-data.sh

```diff
@@ -3,9 +3,14 @@
 # This will make your scripts compatible even if Magisk change its mount point in the future
 MODDIR=${0%/*}
 
-if [[ $(pm list packages "com.google.android.apps.wellbeing") ]]; then
+if [ $(getprop ro.build.version.sdk) -eq 28 ]; then
+  # If PIE
```

Comment:
``# If PIE`` -> ``# If Pie`` (Btw, where did you get the SDK version from? Why is this needed only on Pie?)